SConstruct

```diff
@@ -910,6 +917,17 @@ else:  # GCC, Clang
     if env["werror"]:
         env.Append(CCFLAGS=["-Werror"])
 
+# Configure external includes.
+if env.msvc:
```

Comment:
Seems like this is true for clang-cl, but it needs clang-style `-isystem`.

Reply:
Turns out, that failure was for the same reason as the GCC failure: MSVC cl interprets "external" recursively, but other compilers do not.